syntax = "proto3";

package nvidia.cheminformatics.grpc;

// python -m grpc_tools.protoc -I./grpc/ --python_out=generated --grpc_python_out=generated ./grpc/interpolator.proto
service Interpolator {
  rpc Interpolate (InterpolationSpec) returns (SmilesList) {};
}

message InterpolationSpec {
  string model = 1;
  repeated string smiles = 2;
}

message SmilesList {
  repeated string generatedSmiles = 1;
}
